spec/controllers/events_controller_spec.rb

```diff
@@ -68,6 +98,16 @@
       end
     end
 
+    describe 'PATCH #update' do
+      let(:event) { FactoryGirl.create(:event) }
+      subject { patch :update, params: { id: event.id, event: attr } }
+      let(:attr) { { name: 'New Name' } }
```

Comment:
Super minor but I think it's nice to keep the let statements grouped together. I've kind of been adopting the practice of having the subject above all the let statements just because it's more important than regular variables and we always know where to find it.